ledger/src/blockstore.rs

```diff
@@ -3454,7 +3454,23 @@ fn find_slot_meta_in_cached_state<'a>(
     }
 }
 
-// Chaining based on latest discussion here: https://github.com/solana-labs/solana/pull/2253
+/// For each entry in `working_set` whose did_insert_occur is true, this
+/// function handle its chaining effect by updating the SlotMeta of both
+/// the slot and its parent slot.  In addition, when a slot is newly connected,
```

Comment:
nit: the slot and its parent slot *to reflect that the slot descends from the parent slot*